config.template.js

```diff
@@ -1,13 +1,8 @@
-// copy this to your config file
-// connect to phpmyadmin: https://tools.engr.oregonstate.edu/phpMyAdmin/index.php
-// With your VPN connected, use below user/password to login
-// Server choice: classmysql
-// New table names should start with a_, like a_patients, so they are on the top of the list
-// and distinguishable with the tables I used in cs340
+// USE THIS CREATE YOUR CONFIG FILE
```

Comment:
Can you remove this file from the PR, I've updated it so that it fits a new configuration

Reply:
Not sure how to remove the file from PR, is it OK to just overwrite with the new file?